matplotlib-tests/meta.yaml

```diff
@@ -0,0 +1,18 @@
+package:
+  name: matplotlib-tests
+  version: 1.5.0
+
+source:
+  git_url: https://github.com/matplotlib/matplotlib.git
+  git_rev: v1.5.x
+
+requirements:
+  build:
+    - matplotlib ==1.5.0
```

Comment:
Is there a way to get the version of the current package (on line 3 above) and use that?  Really, the versions have to be the same for this all to work.